xbmc/Application.cpp

```diff
@@ -721,7 +722,7 @@ bool CApplication::CreateGUI()
             info.iHeight,
             info.strMode.c_str());
 
-  g_windowManager.Initialize();
+  m_pGUI.reset(new CGUIComponent());
```

Comment:
to fix the segfault, this should be moved up to the beginning of CApplication::CreateGUI()

Reply:
windowing has to be initialized before gui. windowing must not depend on gui. if this is the case, it needs to be fixed. as an intermediate sollution it can check if CServiveBroker::GetGUI is nullptr before accessing it.

Reply:
@FernetMenta it may also be sufficient to move the ```m_renderGUI = true;``` statement after the m_pGUI initialisation? I need to check that.

Reply:
yes, we should move this statement anyway. good catch

Reply:
popcornmix disabled the splash screen temporarily for RPI nightlies, which is likely related to this, see https://github.com/popcornmix/xbmc/commit/88d06379af004e33c6c5158b0228c1e78a9c9329